README.md

```diff
@@ -169,6 +169,31 @@ The returned `TypeConsistency` is going to be a subset of the following *QoS pol
 
 Note: `TypeConsistency` is an enum with `|` and `&` operators overrided to manage it as a set of QoS policies.
 
+### Module
+Modules are equivalent to C++ namespaces. They can store sets of StructType, Constants and other Modules
+(as submodules).
+They allow organizing types into different scopes, allowing scope solving when accessing the stored types.
+```c++
+Module root;
+Module& submod_A = root.create_submodule("A");
+Module& submod_B = root.create_submodule("B");
+Module& submod_AA = submod_A.create_submodule("A");
+root.set_struct(inner);
```

Comment:
Why set instead of add?
set means for me that you override the previous data with the new one.

Reply:
Changed to just `struct`.